.nycrc

```diff
@@ -2,6 +2,9 @@
 	"extends": "@istanbuljs/nyc-config-typescript",
 	"reporter": ["lcov", "text", "text-summary"],
 	"include": ["src/**"],
+	"exclude": [
+		"src/cli.ts"
```

Comment:
Please check my coment here: https://github.com/SAP/ui5-linter/pull/57#issuecomment-2027064528

I'd appreciate any feedback on this one.